sympy/parsing/autolev/test_examples/test10_in.txt

```diff
@@ -0,0 +1,59 @@
+Newtonian N
```

Comment:
Where do these examples come from? We need to assess the copyright and our use of them. Adding them to the sympy repo means they are licensed under the BSD license.

Reply:
All the examples added till now are from this Autolev Tutorial.
http://web.mae.ufl.edu/~fregly/PDFs/autolev_tutorial.pdf

Reply:
I would like to use some examples from Dynamics Online henceforth but as you said previously we might need to alter the code a little due to copyright issues.

Reply:
Does the tutorial have any copyright information?

Reply:
@certik Do you have any thoughts on how to handle this? I'm pretty sure both of these sources are not under any kind of open license.

Reply:
The first page says Copyright 1996-2005 by Paul Mitiguy and Keith Reckdahl. All Rights Reserved.

Reply:
Yes, regarding the copyright, there are only two ways out. Either

* ask the authors to license the files we use under a BSD license

or

* rewrite the files from scratch (without referencing them), i.e., write them as good tests, without having any similarity to the original files. You can still use the original files on your local computer to ensure it actually works, but in sympy we have to have our own files.